iati_datastore/iatilib/frontend/api1.py

```diff
@@ -67,6 +67,32 @@ def about_dataset(dataset):
             resources=resources,
     )
 
+
+@api.route('/about/datasets/fetch_status')
+def nest_about_dataset():
+    """Output a JSON formatted list of dataset dictionaries containing their resource details.
+    This is an experimental API call and not intended for general use.
```

Comment:
Suggest putting this within a `Warning` scope.

Reply:
done